tools/lib/feature-query.js

```diff
@@ -61,8 +61,8 @@ export class FeatureQuery {
     }
 
     // Filter (potentially) by channel. Choose the most released channel (e.g., 'stable' over 'beta').
-    const bestChannel = q.reduce((channel, spec) => mostReleasedChannel(channel, spec.channel), /** @type {chromeTypes.Channel | undefined} */(undefined));
-    const bestChannelFilter = q.filter(({ channel }) => channel === bestChannel);
+    const bestChannel = q.reduce((channel, spec) => mostReleasedChannel(channel, spec.channel ?? "stable"), /** @type {chromeTypes.Channel | undefined} */(undefined));
+    const bestChannelFilter = q.filter(({ channel }) => channel === bestChannel || (bestChannel === "stable" && typeof channel === "undefined"));
```

Comment:
Good call on the formatting, I just updated that. I think `!channel` would also work, I just try to be explicit with typeof when I can (since sometimes the difference between undefined and false is meaningful).

Reply:
>... sometimes the difference between undefined and false is meaningful

as a language true, though I don't that would be possible in this specific situation. But if you would still rather be specific, is there a reason to to a typeof check rather than an equity check? you declare it as an argument, so it wouldn't be a ReferenceError

Reply:
Yeah, I don't disagree. I think it's just a preference thing - I've updated to `!!channel`.